<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auditoriums</class>
 <widget class="QDialog" name="Auditoriums">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Аудитории</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#aaaaaa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">background-color:#606060</string>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="10" colspan="2">
      <widget class="QPushButton" name="pushButton_update">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10" colspan="2">
      <widget class="QPushButton" name="pushButton_delete">
       <property name="styleSheet">
        <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_namesudi_update">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Название аудитории</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox_delete">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
       </property>
       <property name="placeholderText">
        <string>ID поля</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_capacity_insert">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Вместимость</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7" colspan="2">
      <widget class="QLineEdit" name="lineEdit_equipment_insert">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Да/Нет</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="9">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>446</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_id_insert">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>id аудитории</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_update">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
       </property>
       <property name="placeholderText">
        <string>ID поля</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_numberaudi_update">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="3">
      <widget class="QLineEdit" name="lineEdit_equipment_update">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Да/Нет</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="2">
      <widget class="QLineEdit" name="lineEdit_numberaudi_insert">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Номер аудитории</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10" colspan="2">
      <widget class="QPushButton" name="pushButton_insert">
       <property name="styleSheet">
        <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLineEdit" name="lineEdit_namesudi_insert">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Название аудитории</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_capacity_update">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
       </property>
       <property name="placeholderText">
        <string>Вместимость</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
